webpack.config.client.js

```diff
@@ -29,11 +29,14 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
     },
     output: {
       path: resolvePath('build/public'),
-      filename: 'static/js/[name].[hash:8].js',
+      // need unhashed client bundle when running dev server: https://github.com/jaredpalmer/razzle/tree/master/packages/create-razzle-app/templates/default#how-razzle-works-the-secret-sauce
```

Comment:
Since we are no longer using Razzle, instead of linking to the Razzle documentation, I think we should ensure that we have a commensurate explanation within our own documentation in Simorgh.